.travis.yml

```diff
@@ -32,6 +33,12 @@ jobs:
         on:
           branch: master
         skip_cleanup: true
+      - provider: script
+        script: cd che-theia-factory && npm version -no-git-tag-version "0.0.1-${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" && npm publish --access public
```

Comment:
we should push to another org
it avoids to have something "not being validated" being pushed on official npmjs repo

Reply:
yeah OK but for PR for instance, it is hard to change the org name to test it. or is there a npm command to change the org name automatically ?

Reply:
you could do it before the npm command ? 
we were doing some stuff for some time to the 'forked theia'
https://github.com/theia-demo-plugins/wiptheia/blob/master.2018.06.18b/.travis.yml#L78

Reply:
ok let's do like that but replacing the whole module name rather than just the org
s/@eclipse-che\/theia-factory-extension/@wiptheia\/theia-factory-extension (would avoir replacing dependencies that have the same org) would that work for you ?